src/patterns/addresses/index.md.njk

```diff
@@ -100,9 +100,14 @@ When using an address lookup, you should:
 
 #### Allow different postcode formats
 
-It is easier for users if you accept and ignore unwanted characters. This is better than rejecting the input and telling a user they've not provided a valid postcode.
+It's easier for users if you accept and ignore unwanted characters. This is better than rejecting the input and telling the user they have not provided a valid postcode.
```

Comment:
Should we also recommend removing them, in case that's what some users would prefer to do?

```suggestion
It's easier for users if you accept and ignore, or remove, unwanted characters. This is better than rejecting the input and telling the user they have not provided a valid postcode.
```